docs/articles/documentation/concepts/browsers.md

```diff
@@ -0,0 +1,301 @@
+* Microsoft Edge (legacy and Chromium-based)
+* Mozilla Firefox
+* Safari
+* Google Chrome mobile
+* Safari mobile
+
+> TestCafe supports the latest version of each browser unless specified explicitly.
+
+### Locally Installed Browsers
+
+TestCafe can automatically detect popular browsers installed on the local computer.
+You can use a short name - *browser alias* - to identify these browsers when you launch tests.
+
+The following table lists browsers that can be detected automatically.
+
+Browser                                    | Browser Alias
+------------------------------------------ | -------------------
+Chromium                                   | `chromium`
+Google Chrome                              | `chrome`
+Google Chrome Canary                       | `chrome-canary`
+Internet Explorer                          | `ie`
+Microsoft Edge (legacy and Chromium-based) | `edge`
+Mozilla Firefox                            | `firefox`
+Opera                                      | `opera`
+Safari                                     | `safari`
+
+Call the `testcafe` command with the [--list-browsers](../reference/command-line-interface.md#-b---list-browsers) flag to view the list of all the available browsers.
+
+To run tests in a different local browser, specify the path to the browser executable file.
+
+> To test in Microsoft Edge Chromium and Legacy on the same machine, [install them side-by-side](https://docs.microsoft.com/en-us/DeployEdge/microsoft-edge-sysupdate-access-old-edge). Once they are installed, see the [Microsoft Edge Legacy Support](#microsoft-edge-legacy-support) section below for information on how to enable testing in older versions.
+
+For more information and examples, see:
+
+* Command line: [Local Browsers](../reference/command-line-interface.md#local-browsers)
+* API: [runner.browsers](../reference/api/runner/browsers.md)
+* Configuration file: [browsers](../reference/configuration-file.md#browsers)
+
+### Portable Browsers
+
+To use a portable browser, specify the path to the browser executable file. For more information and examples, see:
+
+* Command line: [Portable Browsers](../../command-line-interface.md#portable-browsers)
+* API: [runner.browsers](../reference/api/runner/browsers.md)
+* Configuration file: [browsers](../reference/configuration-file.md#browsers)
+
+### Browsers on Remote Devices
+
+To run tests on a remote mobile and desktop device, this device must have network access to the TestCafe server.
+
+First, you will need to create a remote browser connection.
+
+* Command line: specify the `remote` *alias* (see [Remote Browsers](../reference/command-line-interface.md#remote-browsers))
+* API: use the [createBrowserConnection](../reference/api/testcafe/createbrowserconnection.md) method
+* Configuration file: [browsers](../reference/configuration-file.md#browsers)
+
+After that, TestCafe will provide a URL to open on the remote device in the browser against which you want to test.
```

Comment:
Can you be more specific than "provide"?

Reply:
The way TestCafe provides the URL depends on how you run tests. We can write something like:

`After that, TestCafe generates a URL to open on the remote device in the browser against which you want to test. This URL is then exposed through the API or displayed in the console.`

Reply:
Can we say:
After that, TestCafe generates a URL to open in the browser to be tested. This URL is then exposed through the API or displayed in the console.

Just checking, but regarding the second sentence, if the URL that was generated is displayed in the console, do we then need to copy this URL from the console to the browser of the remote device? If so, do we need to specify this?

Reply:
Just as here: https://github.com/DevExpress/testcafe/pull/4986#discussion_r410213682

I don't think we can say about 'testing browsers'. TestCafe tests how a website works in this or that browser, as we say 'TestCafe tests a website against this or that browser'.

Reply:
Regarding copying URL - we can specify it but accessing the link through the API is even less trivial. Though, we provide no details regarding this.

We should probably, but for now, we leave it up to the user.

Reply:
Can we then say:
After that, TestCafe generates a URL to open in the browser that you need to test (on a remote device). This URL is then exposed through the API or displayed in the console.